README.md

```diff
@@ -38,50 +41,47 @@ terraform {
   required_providers {
     auth0 = {
       source  = "auth0/auth0"
-      version = ">= 0.34" # Refer to docs for latest version
+      version = ">= 0.43.0" # Refer to docs for latest version
```

Comment:
It'd be nice to have this number bumped when preparing a new release, e.g. in the same PR that updates the Changelog.